cogs/maplist_exp.py

```diff
@@ -0,0 +1,372 @@
+# Std Lib Imports
```

Comment:
whats the difference bettween maplist and maplist_exp?

Reply:
maplist exp was a dev version I had where I was refactoring it so we could have a /upload command and still use the same functions, did not think it was in this pr